app/src/main/java/net/programmierecke/radiodroid2/adapters/ItemAdapterStation.java

```diff
@@ -599,7 +622,23 @@ private void setupCompactStyle(final StationViewHolder holder) {
             holder.imageViewIcon.getLayoutParams().height = (int) getContext().getResources().getDimension(R.dimen.compact_style_icon_height);
         }
     }
-    
+
+    private void setupIconsOnlyStyle(final StationViewHolder holder) {
```

Comment:
Maybe a separate adapter with separate item layout? Because using this adapter for icons only view is too much in my opinion.

Reply:
You are probably right, I also thought so.